src/main/java/com/zendesk/maxwell/producer/MaxwellKafkaProducer.java

```diff
@@ -53,7 +55,26 @@ public void onCompletion(RecordMetadata md, Exception e) {
 	}
 }
 
-public class MaxwellKafkaProducer extends AbstractAsyncProducer {
+
+public class MaxwellKafkaProducer extends AbstractProducer {
+	private final LinkedBlockingQueue<RowMap> queue;
+	private final MaxwellKafkaProducerWorker worker;
+
+	public MaxwellKafkaProducer(MaxwellContext context, Properties kafkaProperties, String kafkaTopic) {
+		super(context);
+		this.queue = new LinkedBlockingQueue<>(100);
```

Comment:
sure.  I can buy that.  I'll switch all the queues to that, according to stackoverflow it's a little less locky-and-gcy